### ISSUE TITLE
Run on goggles startup

### ISSUE THREAD

**Issue by zachsa999** [stylesuxx/dji-hd-fpv-led-matrix#5]
Thanks for the interesting project. How could I feasably run this on goggle startup? is there a file in WTF os that allows this?

**Comment by DemKat** [comment]
> Thanks for the interesting project. How could I feasably run this on goggle startup? is there a file in WTF os that allows this?

Ik it's been a while but it's now 3 am and i figured it out. You could use vim in the CLI to make a file at /opt/ec/init.d/[FILE NAME] (you might want to start the name with S99 so it starts after everything else). This file would just be a shell script so you can do whatever else you want o be done on startup. I used the buzzer test from /system/bin to make it play a startup tune fo rexample

**Comment by KevinKor001** [comment]
yes, basically what @DemKat said... just be sure to make the service executable , and also don't place it directly on the root directory.

Consider this ticket closed?
